README.md

```diff
@@ -81,3 +81,7 @@ The steps are also available in the [PULL_REQUEST_TEMPLATE](/.github/PULL_REQUES
 
 This application is hosted on the MoJ Cloud Platform. For further details
 head over to [our infrastructure documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4325244964/Manage+infrastructure).
+
+## Environments
+
+[Details of the different environments and their roles can be found in Confluence](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4330226204/Environments).
```

Comment:
My preference would be for just "can be found in Confluence" or even just "Confluence" to be the link

Reply:
I think I am happy to keep a sentence here to describe why you might want to dig into this.